samples/apps/forum/src/types/ccf.ts

```diff
@@ -48,6 +48,26 @@ export interface KVMap {
 
 export type KVMaps = { [key: string]: KVMap };
 
+export interface ProofElement {
+  left?: string;
+  right?: string;
+}
+
+export type Proof = ProofElement[];
+
+export interface Receipt {
+  signature: string;
```

Comment:
Could we add a jsdoc comments like /** base64-encoded signature of the root by the node identified by nodeId */, and if we did, would it get picked up by sphinx-js? Just wondering, not requesting :)

Reply:
Yes and yes, see the CryptoKeyPair interface in https://github.com/microsoft/CCF/pull/2313